src/app/src/util/csrfRequest.js

```diff
@@ -9,4 +9,12 @@ const csrfRequest = axios.create({
     },
 });
 
+// Not related to CSRF protection, but this is the appropriate place to set the
```

Comment:
Not required, but this may be an opportunity to rename this module to be something more generic (`apiRequest` instead of `csrfRequest`). That would require changes throughout the codebase though, which what this is trying to avoid.

Reply:
After some reflection I do think it probably worth it to give this module a more correct name rather than adding the comment. While it will create some "blame noise" it is a good idea to avoid a potentially misleading name.

Reply:
Renamed in 2854f09